test/unit/app/browser/api/ledgerTest.js

```diff
@@ -2169,16 +2171,47 @@ describe('ledger api unit tests', function () {
     after(function () {
       checkForUpdateSpy.restore()
       roundtripSpy.restore()
+      fakeClock.restore()
     })
 
     it('null case', function () {
       ledgerApi.checkReferralActivity(defaultAppState)
+      assert(roundtripSpy.notCalled)
+      assert(checkForUpdateSpy.withArgs(false, true).calledOnce)
+    })
+
+    it('counter is greater then 30', function () {
+      const state = defaultAppState
+        .setIn(['updates', 'referralDownloadId'], 1234)
+        .setIn(['updates', 'referralAttemptCount'], 30)
+      ledgerApi.checkReferralActivity(state)
       assert(checkForUpdateSpy.withArgs(false, true).calledOnce)
+      assert(roundtripSpy.notCalled)
+    })
+
+    it('first attempt', function () {
```

Comment:
might be good to change what this test is named, so it matches the `it` style.  For example:
`it('makes call if this is the first attempt', function () {`

and below:
`it('does not make call if within 24 hours of an attempt', function () {`

Reply:
done